components/brave_rewards/resources/page/components/pageWallet.tsx

```diff
@@ -480,15 +480,6 @@ class PageWallet extends React.Component<Props, State> {
     )
   }
 
-  getInternalFunds = () => {
```

Comment:
I was waiting for this change, as now we can pass the balance as an optional `double`! 🎉
Do you feel like resolving this in this PR by any chance? 🙂

Reply:
Actually, never mind, I'll address this in another PR coming up soon.

Reply:
Sure, no worries.